p2p/transport/quicreuse/connmgr.go

```diff
@@ -25,19 +25,21 @@ type ConnManager struct {
 	quicListenersMu sync.Mutex
 	quicListeners   map[string]quicListenerEntry
 
-	srk quic.StatelessResetKey
+	srk      quic.StatelessResetKey
+	tokenKey quic.TokenGeneratorKey
 }
 
 type quicListenerEntry struct {
 	refCount int
 	ln       *quicListener
 }
 
-func NewConnManager(statelessResetKey quic.StatelessResetKey, opts ...Option) (*ConnManager, error) {
+func NewConnManager(statelessResetKey quic.StatelessResetKey, tokenKey quic.TokenGeneratorKey, opts ...Option) (*ConnManager, error) {
 	cm := &ConnManager{
 		enableReuseport: true,
 		quicListeners:   make(map[string]quicListenerEntry),
 		srk:             statelessResetKey,
+		tokenKey:        tokenKey,
 	}
```

Comment:
This needs to be passed to the actual quic.Transport instance, right? Like we pass stateless reset key here
https://github.com/libp2p/go-libp2p/blob/99f761127098f1a1cac564d79955cf66c1665a20/p2p/transport/quicreuse/connmgr.go#L200

```
	return &singleOwnerTransport{Transport: quic.Transport{Conn: conn, StatelessResetKey: &c.srk}, packetConn: conn}, nil

```

Reply:
Good catch! Fixed in 3a6d0ee74fbf168df70da4ba6491a1ed910e6090.